ts/sveltelib/export-runtime.ts

```diff
@@ -6,7 +6,9 @@
 // would not work.
 
 import * as svelteRuntime from "svelte/internal";
+import * as svelteStore from "svelte/store";
 
 import { registerPackageRaw } from "../lib/runtime-require";
 
 registerPackageRaw("svelte/internal", svelteRuntime);
+registerPackageRaw("svelte/store", svelteStore);
```

Comment:
This change is unrelated to the PR, but it's quite helpful to debugging the UI in the webview with a quick `require('anki/store').get(ourStore)`